src/pageObjects/GaragePage/GaragePage.js

```diff
@@ -0,0 +1,31 @@
+import BaseComponent from '../../components/BaseCoomponent';
+
+export default class GaragePage extends BaseComponent {
+  _garagePageContentSelector = 'div.app-content';
+
+  _settingsBtnSelector = 'a[routerLink=settings]';
+
+  _removePopupSelector = 'app-remove-account-modal';
+
+  _removeMyAccountBtnSelector = 'button:has-text("Remove my account")';
+
+  _removeButtonSelector = ('div.modal-footer button:has-text("Remove")');
```

Comment:
it is not related to GaragePage, It's on SettingsPage

Reply:
Created SettingsPage pom and updated GaragePage pom